examples/gpu-sharing/config.yaml

```diff
@@ -0,0 +1,36 @@
+bundled_packages_dir: packages
+data_dir: data
+description: |
+  This is an example of a truss that shares GPU across multiple models. This truss serves
+  three models: a text to image model, an image to image model and an in-painting mode.
+  Specific model can be targeted by providing `model_sub_name` in the request dictionary. Please
+  refers to examples.yaml for examples.
+  A model is loaded into GPU memory when needed. When a different model is invoked, then the previous
+  one is offloaded and replaced with the new one. All models remain in the regular memory though, so that
+  moving them to GPU memory later is faster.
+environment_variables: {}
+examples_filename: examples.yaml
+external_package_dirs: []
```

Comment:
instead of putting all the models code here, should this example use `external_package_dirs` that point at the trusses that should be included?

Reply:
I think the whole separate trusses might give the wrong impression, we can't really support arbitrary trusses. The trusses have to match in python and system requirements and more. I feel this example is close enough to how someone would manually do this. We have other options for sharing GPU across arbitrary trusses that would be better. e.g. we could provide a command line command that could check different trusses for compatibility and generate a conbination truss, perhaps it was ask users to resolve conflicts that are resolvable. Supporting arbitrary trusses needs more thought.